### PR TITLE
feat: uptake field value v2

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue Overview

This PR addresses #3396

- [x] This change addresses the issue in full

## Description

This PR uptakes the new FieldValue component from seeds.
[ZeroHeight Docs](https://zeroheight.com/5e69dd4e1/p/715019-field-value/b/93a6d7)
[Storybook](https://storybook-ui-seeds.netlify.app/i)

## How Can This Be Tested/Reviewed?

As compared to dev, the styling is subtly different intentionally in terms of the font size, color, and padding.

To test this PR, check out the new component in the following places:
* Partners listing detail page
* Partners application detail page
* Public application add household members page
* Public application summary page

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/partners/styles/overrides.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+:root {
+  .field-value-error {
```
Comment (emilyjablonski) [inline sites/partners/styles/overrides.scss:2]: This is a temporary workaround. We have this issue where some of our `Select` and `FieldGroup` components are using the old `ViewItem` as that field(s) label. When a field is in an error state, the label also needs to turn red. It was a misuse of the component. Rather that create a new one for this interim period before those fields are in seeds, I'm just adding the error state here.

**Comment by ludtkemorgan** [comment]
The unit tests should be fixed with this change in ui-seeds: https://github.com/bloom-housing/ui-seeds/pull/49

**Review by ludtkemorgan — COMMENTED** [review]
Code wise this looks good. Are the failing cypress tests related to these changes? Looks like it is on a part of the code that you did touch.

**Comment by emilyjablonski** [comment]
@ludtkemorgan Yep it's blocked by this seeds PR: https://github.com/bloom-housing/ui-seeds/pull/53

**Review by ludtkemorgan — APPROVED** [review]
🏷️

**Review by ColinBuyck — APPROVED** [review]
LGTM 🪀 really like the subtle visual changes too!

**Pull request merged**